test/e2e/view-caseload.js

```diff
@@ -64,21 +64,21 @@ describe('View your caseload flow', () => {
     it('with the correct tabs and tables', () => {
       return browser.url(teamDefaultUrl + '/caseload')
         .waitForExist('[href="#overall"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#overall-enhanced')
         .waitForExist('[href="' + workloadOwnerDefaultUrl)
         .click('[href="#custody"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#custody-enhanced')
         .click('[href="#community"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#community-enhanced')
         .click('[href="#license"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#license-enhanced')
```

Comment:
This is just checking the tabs, not the tables as the test name suggests?

Reply:
Added check for tables aswell